keepassxc-browser/background/event.js

```diff
@@ -204,8 +204,8 @@ kpxcEvent.pageClearLogins = function(tab, alreadyCalled) {
     return Promise.resolve();
 };
 
-kpxcEvent.pageGetLoginId = function() {
-    return Promise.resolve(page.loginId);
+kpxcEvent.pageGetLoginId = async function() {
+    return page.loginId;
```

Comment:
Do we need an async function for this? Seems like a static data member access.

Reply:
Well, not needed but async functions return Promise automatically, so it's a bit more clear to read.